tests/annotation_snpeff.nf.test

```diff
@@ -5,53 +5,6 @@ nextflow_pipeline {
     tag "pipeline"
     tag "pipeline_rnavar"
 
-    test("-profile test --tools snpeff --download_cache") {
```

Comment:
Why removal of this test?

Reply:
it's currently failing because of some underlying issues on the snpeff side.
So I figured it was a good time to remove because it's not actually that useful and we can use annotation-cache for this since it has been downloaded already